woocommerce/woocommerce
# Add greater specificity to required styling

Adds extra selectors to required and optional form styling.

Closes #20295

### How to test the changes in this Pull Request:

1. 2017 theme
2. Check required * are displayed on checkout
3. Hide them via customiser. Ensure hidden, and optional is displayed instead.
